rs/ethereum/ledger-suite-orchestrator/src/scheduler/mod.rs

```diff
@@ -932,8 +932,6 @@ fn icrc1_ledger_init_arg(
         ),
         max_memo_length: Some(MAX_MEMO_LENGTH),
         feature_flags: Some(ICRC2_FEATURE),
-        maximum_number_of_accounts: None,
-        accounts_overflow_trim_quantity: None,
```

Comment:
@maciejdfinity : Please add a PR description, being mindful that those PRs can be reviewed by the community when verifying proposals.

1. What's the general motivation for the change
2. What's the rough high level idea, any noteworthy implementation details?
3. I feel that the reason for removing fields from the init args should be explained.

Reply:
Good point! I added a description!